### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by manuelDeLaEranueva**

### CODE REVIEW

File: staff/manuel-eranueva/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# BASH
+
+Command-line interface commands typical in BASH.
+
+## pwd - path to working directory
+
+```para saber dónde estás.```
```
Comment (manuelbarzi) [inline staff/manuel-eranueva/bash/README.md]: move comments out of the code block section. and add demo code inside block sections, instead.

File: staff/manuel-eranueva/bash/README.md
Comment (260677) [inline staff/manuel-eranueva/bash/README.md:1]: looking great, keep up!!

File: staff/manuel-eranueva/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+$ cd "direcotry-name"
+```
+
+## mv - move
+
+```sh
+$ mv document1.js files1/document1.js
+```
+
+```
```
Comment (manuelbarzi) [inline staff/manuel-eranueva/bash/README.md]: specify `sh` language in code block

Comment (manuelDeLaEranueva) [reply staff/manuel-eranueva/bash/README.md]: Done.

**Pull request merged**